go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/nanmu42/etherscan-api
 
-go 1.13
+go 1.19
 
-require github.com/google/go-cmp v0.5.7 // indirect
+require github.com/google/go-cmp v0.5.7
+
+replace github.com/nanmu42/etherscan-api v1.9.0 => github.com/Pashteto/etherscan-api v1.0.0
```

Comment:
This replace command does not look right...

Reply:
i don't want to add the replace line into your main repo. I just wanted to get a link to the forked, edited package into my project and unfortunately it wasn't working.
I can revert to the commit cf99fda6d3e7b8c40e2f2e93ed29499c0fed52af if you want to merge these changes.